Make failed on macOS

**Describe the bug**

Following the [Dev Guide](https://github.com/istio/istio/wiki/Dev-Guide#building-the-code), `make` failed with the following details
```
2018-08-28T06:18:08.827058Z	info	skipping cluster registries, no kube-client created
2018-08-28T06:18:08.827134Z	info	Adding Config registry adapter
2018-08-28T06:18:08.827370Z	info	ads	Starting ADS server with rateLimiter=10 burst=100
2018-08-28T06:18:08.827649Z	info	Discovery service started at http=[::]:57075 grpc=[::]:57076
Listening HTTP/1.1 on 57079
Listening HTTP/1.1 on 57080
Listening HTTP/1.1 on 57081
Listening HTTP/1.1 on 57082
Listening GRPC on 57083
Listening GRPC on 57084
PARSE ERROR: Argument: --log-format
             Couldn't find match for argument

Brief USAGE:
   GOPATH/out/darwin_amd64/release/envoy
                                        [--disable-hot-restart]
                                        [--max-obj-name-len <uint64_t>]
                                        [--max-stats <uint64_t>] [--mode
                                        <string>] [--parent-shutdown-time-s
                                        <uint32_t>] [--drain-time-s
                                        <uint32_t>]
                                        [--file-flush-interval-msec
                                        <uint32_t>] [--service-zone
                                        <string>] [--service-node <string>]
                                        [--service-cluster <string>]
                                        [--hot-restart-version]
                                        [--restart-epoch <uint32_t>]
                                        [--log-path <string>] [-l <string>]
                                        [--local-address-ip-version
                                        <string>] [--admin-address-path
                                        <string>] [--v2-config-only] [-c
                                        <string>] [--concurrency
                                        <uint32_t>] [--base-id <uint32_t>]
                                        [--] [--version] [-h]

For complete USAGE and HELP type:
   GOPATH/out/darwin_amd64/release/envoy --help

--- FAIL: TestAgent (10.05s)
	agent_test.go:90: failed to start envoy after 10s. Error: Get http://127.0.0.1:57086/server_info: dial tcp 127.0.0.1:57086: connect: connection refused
FAIL
FAIL	istio.io/istio/pkg/test/framework/components/apps/local/envoy/agent/pilot	10.095s
```

I'm doing it on macOS.

Branch:
```
$ git branch
* (HEAD detached at origin/master)
  collab-gcp-identity
  master
```

Most recent git commitId: 7904b4d58844582f4e87189020938b840bc0f378

--- a/bin/init.sh
+++ b/bin/init.sh
@@ -98,7 +98,7 @@ ISTIO_ENVOY_VERSION=${ISTIO_ENVOY_VERSION:-${PROXY_REPO_SHA}}
 ISTIO_ENVOY_DEBUG_URL=${ISTIO_ENVOY_DEBUG_URL:-https://storage.googleapis.com/istio-build/proxy/envoy-debug-${ISTIO_ENVOY_VERSION}.tar.gz}
 ISTIO_ENVOY_RELEASE_URL=${ISTIO_ENVOY_RELEASE_URL:-https://storage.googleapis.com/istio-build/proxy/envoy-alpha-${ISTIO_ENVOY_VERSION}.tar.gz}
 # TODO Change url when official envoy release for MAC is available
-ISTIO_ENVOY_MAC_RELEASE_URL=${ISTIO_ENVOY_MAC_RELEASE_URL:-https://storage.googleapis.com/istio-on-macos/releases/0.7.1/istio-proxy-0.7.1-macos.tar.gz}
+ISTIO_ENVOY_MAC_RELEASE_URL=${ISTIO_ENVOY_MAC_RELEASE_URL:-https://github.com/istio/proxy/releases/download/1.0.2/istio-proxy-1.0.2-macos.tar.gz}
 
 # Normally set by the Makefile.
 # Variables for the extracted debug/release Envoy artifacts.

